test/com/moclojer/rq/utils_test.clj

```diff
@@ -4,12 +4,14 @@
    [com.moclojer.rq.utils :as utils]))
 
 (t/deftest pattern->str-test
+  (t/testing "packing"
   [(t/is "my-queue" (utils/pack-pattern :none "my-queue"))
```

Comment:
@Felipe-gsilva have you formatted this?

Reply:
just added this testing tag, no changes at all on formatting